source/content/addons/object-cache/howto/drupal.md

```diff
@@ -119,6 +119,26 @@ contributors: [cityofoaksdesign, carolynshannon, jms-pantheon, whitneymeredith]
 
    </Alert>
 
+   <Accordion title="Database Cleanup (recommended)" id="database-cleanup-drupal" icon="lightbulb">
+   
+   After enabling Redis, there are cache tables in the database that are no longer being used. Even when the Drupal cache is cleared, these tables will not be emptied. For sites that were live for awhile before Redis was enabled, there could be significant amounts of data in these tables. Removing this data could increase the speed of cloning, exporting, and backing up the database.
```

Comment:
@ccharlton are these tables not emptied by regular cache clears? I see the logic in emptying them, but I'm surprised we need to specify direct `sql` queries.

Reply:
There are instances where stale data has snagged customers on setup. Moving the tip to recommended but not required seemed more appropriate than leaving it as optional.

Reply:
But does the stale data get removed with a cache clear? I would rather recommending to customer that they click the cache clear button than they figure out (perhaps for the first time) how to directly run mysql commands against Pantheon.

Reply:
@ccharlton, can you respond here?

Reply:
@stevector and @ccharlton after Redis is enabled on a site, truncating the cache tables / bins that have moved to Redis storage is a required step and is not automatically done by enabling the Redis module, running drush cr, or clicking the cache clear button in the Pantheon dashboard. This is currently not documented in a stable release of the Redis module: https://www.drupal.org/project/redis/issues/3177375

If this step is skipped once Redis is enabled, the result is race conditions when invalidating cache which are incredibly difficult to diagnose and can result in very strange issues on the site - eg config appears to revert to old values after running a drush config:import after a deployment. The most failsafe approach is to just truncate all cache* tables which can be done with a direct mysql query, or via drush: 

```
# Get a list of all cache tables
CACHETABLES="$(terminus drush <site>.<env> -- sql:query "SHOW TABLES LIKE 'cache%';")"

# Trucate each cache table in a loop to avoid resource contention and potential deadlocks.

for table in $CACHETABLES; do
    terminus drush <site>.<env> -- sql:query "TRUNCATE TABLE $table;"
done
```